src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/scenario/JiraCoreScenario.kt

```diff
@@ -78,7 +78,7 @@ class JiraCoreScenario constructor() : Scenario {
             searchWithJql to 20,
             viewIssue to 55,
             projectSummary to 5,
-            viewDashboard to 10,
+            viewDashboard to 15,
```

Comment:
It's a good practice to explain the rationale behind such change in a commit description. It will give more context to future maintainers.

Reply:
I will add a comment instead